docs/builders.rst

```diff
@@ -163,3 +163,21 @@ or
 .. hint::
 
     As an alternative, you can set the config option :ref:`needs_build_needumls` to export the needumls files during each build.
+
+
+.. _needs_id_builder:
+
+needs_id
+--------
+.. versionadded:: 1.3.0
+
+The **needs_id** builder exports all found needs and selected filter results to a set json files of each need with the name is ``id`` of need.
+
```

Comment:
In large projects this will creates 1000s of json files. Could this lead to some performance problem due to the IO operations? Are some optimizations thought about for this?

Reply:
> In large projects this will creates 1000s of json files. Could this lead to some performance problem due to the IO operations? Are some optimizations thought about for this?
we need ideas from @danwos  and @r-o-b-e-r-t-o

Reply:
Maybe yes (a little), but it reduces IO/network problems if the data is used via JS-code in the browser (see comment below).
Also you are not forced to use it. And there is another PR, which creates a needs.json per docname.